apmpackage/apm/0.1.0/manifest.yml

```
@@ -0,0 +1,29 @@
+format_version: 1.0.0
+name: apm
+title: Elastic APM
+version: 0.1.0
```

```suggestion
version: 7.11.0
```

?

I expect we'll want to keep the package version in sync with the stack.

---

no other integration follows the stack version  (not even endpoint), any reason we should be the odd one out?

---

Fair enough. No reason I suppose - I just wasn't aware.

---

btw don't know the reason for that exactly, but i assume it is because in most releases there won't be new fields, which would result in many (most?) package versions actually identical.